### ISSUE TITLE
[FEATURE] Add an simple script js file

### ISSUE THREAD

**Issue by vinimarcili** [compodoc/compodoc#1379]
##### **Overview of the feature**

Add one or an folder of js files, that will be includes in all pages with a script tag

##### **Motivation for or Use Case**

I want to add a script in all pages to change body class. Ai have a custom css, and I need to change in some pages the body class.

##### **Related issues**

I don't found any related issues

**Comment by vogloblinsky** [comment]
Did you try this option ? ```-y, --extTheme [file]```

**Comment by vinimarcili** [comment]
Yes, I tried and it works. But changing all the HTML and copying the same "original" code just to add a js script to the page seemed too much to me. An array option just to add any scripts looks good, but it's just an idea
